crates/uv/tests/pip_compile.rs

```diff
@@ -1272,15 +1272,13 @@ fn conflicting_repeated_url_dependency_version_mismatch() -> Result<()> {
     uv_snapshot!(context.compile()
             .arg("requirements.in"), @r###"
     success: false
-    exit_code: 1
+    exit_code: 2
     ----- stdout -----
 
     ----- stderr -----
-      × No solution found when resolving dependencies:
-      ╰─▶ your requirements cannot be used because there are conflicting URLs for
-          package `werkzeug`:
-          - https://files.pythonhosted.org/packages/bd/24/11c3ea5a7e866bf2d97f0501d0b4b1c9bbeade102bb4b588f0d2919a5212/Werkzeug-2.0.1-py3-none-any.whl
-          - https://files.pythonhosted.org/packages/ff/1d/960bb4017c68674a1cb099534840f18d3def3ce44aed12b5ed8b78e0153e/Werkzeug-2.0.0-py3-none-any.whl
+    error: There are conflicting URLs for package `werkzeug`:
+    - https://files.pythonhosted.org/packages/bd/24/11c3ea5a7e866bf2d97f0501d0b4b1c9bbeade102bb4b588f0d2919a5212/Werkzeug-2.0.1-py3-none-any.whl
+    - https://files.pythonhosted.org/packages/ff/1d/960bb4017c68674a1cb099534840f18d3def3ce44aed12b5ed8b78e0153e/Werkzeug-2.0.0-py3-none-any.whl
```

Comment:
Is this an improvement? Should we clarify that the URLs come from the provided requirements?